examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -1,5 +1,21 @@
 # Matter nRF Connect All Clusters Example Application
 
+> **Note:** This example is intended only to perform smoke tests of a Matter
+> solution integrated with nRF Connect SDK platform. The example quality is not
+> production ready and it may contain minor bugs or use not optimal
+> configuration. It is not recommended to use this example as a basis for
+> creating a market ready product.
```

Comment:
As a basis .. reads to me like as a starting point. Isn't that the whole point of this SDK - to act as a starting point?

Reply:
Maybe this should be called a reference? Well, this actually depends how the user is going to understand the reference word :smile: What I would like to achieve is inform someone that "Hey you can use this example to play with the platform, or even create an own app, but please be careful, as this is not tested enough to call it production ready. If you want to build an own product, I recommend to use samples from NCS as a reference.". Does it sound right to you, or I should try to use other wording to not make it confusing?